types/selenium-webdriver/package.json

```diff
@@ -59,6 +60,10 @@
         {
             "name": "Shubh Sheth",
             "githubUsername": "shubhsheth"
+        },
+        {
+            "name": "Sang Nguyen",
+            "githubUsername": "sangcnguyen"
```

Comment:
Are you intentionally modifying this to become an owner? This is not needed for credit, but is instead how we managed github pings/reviews.

Reply:
Now removed, I’d like to keep this package up-to-date with the upstream repository instead of owning it.